Backend/config/unittest.cfg

```diff
@@ -1,2 +1,13 @@
 [unittest]
-start-dir = Backend
\ No newline at end of file
+start-dir = .
+[coverage]
+always-on = True
+coverage-report = html
+coverage =
+  mentii
+  utils
+  problems
+omit =
+  mentii/__init__.py
+  utils/__init__.py
+  problems/__init__.py
```

Comment:
what about the virtual environment in the problems folder? Also I'm guessing every time the tests are run the coverage folder is overwritten?